modules/cco-ccoctl-upgrading.adoc

```diff
@@ -42,87 +38,87 @@ $ oc get secret bound-service-account-signing-key \
 . Use the `ccoctl` tool to process all `CredentialsRequest` objects by running the command for your cloud provider. The following commands process `CredentialsRequest` objects:
 +
 .Amazon Web Services (AWS)
-[%collapsible]
-====
 [source,terminal]
 ----
-$ ccoctl aws create-all \// <1>
-  --name=<name> \// <2>
-  --region=<aws_region> \// <3>
-  --credentials-requests-dir=<path_to_credentials_requests_directory> \// <4>
-  --output-dir=<path_to_ccoctl_output_dir> \// <5>
-  --public-key-file=<path_to_ccoctl_output_dir>/serviceaccount-signer.public \// <6>
-  --create-private-s3-bucket <7>
-----
-<1> To create the AWS resources individually, use the "Creating AWS resources individually" procedure in the "Installing a cluster on AWS with customizations" content. This option might be useful if you need to review the JSON files that the `ccoctl` tool creates before modifying AWS resources, or if the process the `ccoctl` tool uses to create AWS resources automatically does not meet the requirements of your organization.
-<2> Specify the name used to tag any cloud resources that are created for tracking.
-<3> Specify the AWS region in which cloud resources will be created.
-<4> Specify the directory containing the files for the component `CredentialsRequest` objects.
-<5> Specify the path to the output directory.
-<6> Specify the path to the `serviceaccount-signer.public` file that you extracted from the cluster.
-<7> Optional: By default, the `ccoctl` utility stores the OpenID Connect (OIDC) configuration files in a public S3 bucket and uses the S3 URL as the public OIDC endpoint. To store the OIDC configuration in a private S3 bucket that is accessed by the IAM identity provider through a public CloudFront distribution URL instead, use the `--create-private-s3-bucket` parameter.
-====
+$ ccoctl aws create-all \
+  --name=<name> \//
+  --region=<aws_region> \
+  --credentials-requests-dir=<path_to_credentials_requests_directory> \
+  --output-dir=<path_to_ccoctl_output_dir> \
+  --public-key-file=<path_to_ccoctl_output_dir>/serviceaccount-signer.public \
+  --create-private-s3-bucket
+----
+where:
```

Comment:
```suggestion
+
where:
```
You'll need the `+` in these usually, to hook the notes to the steps; question--can these go in a table in a reference module?